code/modules/mob/living/carbon/human/species_types/synths.dm

```diff
@@ -1,142 +0,0 @@
-/datum/species/synth
-	name = "Synth" //inherited from the real species, for health scanners and things
-	id = SPECIES_SYNTH
-	say_mod = "beep boops" //inherited from a user's real species
-	sexes = 0
```

Comment:
Any particular reason for this? It should be noted in the CL otherwise, I know some fugitive events use synths

Reply:
Synths are borderline unused shitcode that hasn't even worked properly in years, and maintaining them was more effort than it is worth I.M.O